docs/License-Guidelines.md

```diff
@@ -0,0 +1,104 @@
+# License Guidelines
+
+We only accept formulae that use a [Debian Free Software Guidelines license](https://wiki.debian.org/DFSGLicenses) or are released into the public domain following [DFSG Guidelines on Public Domain software](https://wiki.debian.org/DFSGLicenses#Public_Domain).
+
+## Specifying a License
+
+All licenses are identified by their license identifier from the [SPDX License List](https://spdx.org/licenses/).
+
+Specify a license by passing it to the `license` method:
+
+```ruby
+license "MIT"
+```
+
+The public domain can be indicated using a symbol:
+
+```ruby
+license :public_domain
+```
+
+## Complex SPDX License Expressions
+
+Some formulae have multiple licenses that need to be combined in different ways. In these cases, a more complex license expression can be used. These expressions are based on the [SPDX License Expression Guidelines](https://spdx.github.io/spdx-spec/appendix-IV-SPDX-license-expressions/).
+
+Add a `+` to indicate that the user can choose a later version of the same license (this doesn't apply to GNU licenses like `GPL`, `LGPL`, `AGPL` and `GFDL` which require either `-only` or `-or-later`):
```

Comment:
I think information in `()` is rather significant to be placed in `()`. Perhaps, this could be moved to after the code sample below but before `license "GPL-1.0-or-later"` example

Reply:
Thanks! Good suggestion!